eks-hybrid-iam-ra-setup.sh

```diff
@@ -3,14 +3,21 @@
 exec >&2
 set -eu -o pipefail
 
-declare -r SECRETS_DIR="/.bottlerocket/rootfs/root/.aws"
+declare -r HOST_ROOTFS="/.bottlerocket/rootfs"
+declare -r SECRETS_DIR="${HOST_ROOTFS}/root/.aws"
+declare -r SIGNING_HELPER_SERVICE_TEMPLATE_PATH="/usr/share/aws_signing_helper_update_service.in"
```

Comment:
Let's put this in a subdirectory - `/usr/share` is meant to be organized by the program that owns the data files there.
```suggestion
declare -r SIGNING_HELPER_SERVICE_TEMPLATE_PATH="/usr/share/bootstrap/aws_signing_helper_update_service.in"
```

Reply:
Done